hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestService.java

```diff
@@ -90,7 +89,7 @@ public HiveServer2 start() throws IOException {
     Objects.requireNonNull(workDir, "The work dir must be set before starting cluster.");
 
     if (hadoopConf == null) {
-      hadoopConf = HoodieTestUtils.getDefaultHadoopConf();
+      hadoopConf = new Configuration();
```

Comment:
Why does this have to change

Reply:
This was probably a side effect of merging, fixed it.